.gitpod.yml

```diff
@@ -0,0 +1,11 @@
+image:
+  file: .gitpod.Dockerfile
+tasks:
+  - init: make dev-gitpod
+vscode:
+  extensions:
```

Comment:
Don't recall if they were ported to the open vscode extension, but equally helpful for everyone starting:

- oderwat.indent-rainbow
- yzhang.markdown-all-in-one  # autoformat, better visualization, snippets, and markdown export to multiple fmts
- bierner.markdown-mermaid # previews mermaid diagrams when previewing markdown
- matangover.mypy # highlight mypy issues
- njpwerner.autodocstring # auto-generate docsstrings in numpy format that we use
- netcorext.uuid-generator # for those helping create code snippets for docs
- streetsidesoftware.code-spell-checker # spell checker that works with camel case too
- bungcip.better-toml # in case GitPod doesn't have support for TOML `pyproject.toml`